release-notes/changes-in-timescaledb-2.md

```diff
@@ -320,6 +320,40 @@ After upgrading to TimescaleDB 2.0, **retention policies will no longer fail due
 continuous aggregates** and users have to ensure that retention and continuous aggregate policies have the 
 desired interplay. 
 
+Another change in 2.0 is that `drop_chunks` and the retention policy do not refresh continuous 
+aggregates on the dropped chunks. Previously dropping a chunk would call a refresh on each 
+continuous aggregate to recompute it according to existing invalidations. The refresh was 
+unlikely to include all updates due to `ignore_invalidations_older_than`, which prevents to 
+record invalidations for updates older than the threshold. If a user carefully crafted 
+parameters to the retention policy and the continuous aggregate that no update is missed, 
+the refresh could include previously dropped values leading to unexpected outliers in 
+aggregated values. TimescaleDB 2.0 introduces new function 
+`_timescaledb_internal.refresh_continuous_aggregate`, which refreshes 
+exactly a chunk, and can be used together with drop_chunks. For example, following action can 
+be defined:
+
+```sql
+CREATE OR REPLACE PROCEDURE refresh_and_drop_policy(job_id int, config jsonb) LANGUAGE PLPGSQL AS
+$$
+DECLARE
+  drop_after interval;
+  hypertable text;
+BEGIN
+  SELECT jsonb_object_field_text(config, 'drop_after')::interval
+    INTO STRICT drop_after;
+  SELECT jsonb_object_field_text(config, 'hypertable')
+    INTO STRICT hypertable;
+  BEGIN
+    SELECT refresh_chunks(cagg, show_chunks(older_than => drop_after))
```

Comment:
I think you mean this:

```suggestion
    SELECT _timescaledb_internal.refresh_continuous_aggregate(cagg, show_chunks(older_than => drop_after))
```

Reply:
👍  Thank you for finding! I missed to fix this one after copying from the design doc.

Reply:
Thank you! Fixed